src/components/ScholarshipDetailCard.jsx

```diff
@@ -1,29 +1,112 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Box, Link, makeStyles, Typography } from '@material-ui/core';
+import {
+  Button,
+  Box,
+  Link,
+  makeStyles,
+  Typography,
+  Chip,
+} from '@material-ui/core';
+import { Share, Send, Info } from '@material-ui/icons';
 import ScholarshipAmount from '../types/ScholarshipAmount';
+import ScholarshipEligibility from '../types/ScholarshipEligibility';
 
-const useStyles = makeStyles(() => ({
+import ScholarshiDetailCardCell from './ScholarshipDetailCardCell';
+
+const useStyles = makeStyles((theme) => ({
+  name: {
+    fontWeight: '500',
+  },
+  actionsWrapper: {
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(4),
+    padding: `${theme.spacing(1)}px 0`,
+  },
+  applyBtn: {
+    backgroundColor: '#3FB1B5',
```

Comment:
I don't think this is necessary? You applied `color="primary"` to this button so it should pick up the primary theme color.